press/press/doctype/deploy_candidate/deploy_candidate.py

```diff
@@ -403,7 +406,9 @@ def _run_docker_build(self, no_cache=False):
 			f"{settings.docker_registry_url}/{namespace}/{self.group}"
 		)
 
-		self.docker_image_tag = self.name
+		self.docker_image_tag = (
+			self.name + "-arm" if "arm" in self.architecture else self.name
```

Comment:
Why not add suffix regardless of arch? General cases are always better than special cases. There might be 3rd arch someday.